website/sync-community-repos.js

```diff
@@ -74,54 +57,233 @@ async function graphqlQuery(after = null) {
 }
 
 async function collectRepos() {
-  const repos = [];
-
-  let endCursor = null;
-  do {
-    const json = await graphqlQuery(endCursor);
-
-    const {repositories} = json.data.organization;
-    repos.push(...repositories.nodes);
-
-    const {pageInfo} = repositories;
-    endCursor = pageInfo.endCursor;
-  } while (endCursor != null);
-
-  return repos;
+  const repos = [
+    {
+      owner: 'react-native-community',
+      name: 'react-native-webview',
+      type: 'Component',
+      title: 'Webview',
+    },
+    {
+      owner: 'react-native-community',
+      name: 'react-native-cameraroll',
+      type: 'API',
+      title: 'CameraRoll',
+    },
+    {
+      owner: 'harisbaig100',
```

Comment:
What's the `owner` role here? Just asking as some of them have `react-native-community`.

Reply:
that's used to fetch the repo details from GitHub, not for display